docs/source/configuration/slots.md

```diff
@@ -0,0 +1,89 @@
+# Slots
+
+Slots are insertion points in the Volto rendering tree structure.
+This concept is inherited from Plone ClassicUI artifact `Viewlets`.
+
+## Anatomy
+
+Slots are named, and they can contain a list of different slot components.
+Slots components are also named, and they are registered in the configuration registry using a specific API for slots.
+The main trait of a slot component is that its render are controlled by a list of conditions (predicates).
+Multiple slot components can be registered under the same name, as long as they have different predicates.
+
+Slot (eg. `toolbar`)
+  - SlotComponent (eg. `edit`)
+    - predicates (eg. only appear in `/de/about`)
+    - predicates (eg. only appear if content type is one of `Document` or `News Item`)
+    - no predicates (eg. always appear)
+  - SlotComponent (eg. `contents`)
+  - SlotComponent (eg. `more`)
+
+The order in which the components should render is governed by the order that they were registered.
+You can change the order of the defined slot components for a different slot using the API. (pending)
+You can even delete the rendering of a registered slot component using the API (pending)
```

Comment:
Let's wrap the above two lines in an admonition.

````
```{todo}
You can change the order of the defined slot components for a different slot using the API.
You can even delete the rendering of a registered slot component using the API.
```
````

Reply:
I will continue working on this.